samples/rest/BingSpellCheckV7.py

```diff
@@ -1,53 +1,51 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why are we removing the copyright comment

Reply:
Copyright headers are an org requirement.

Reply:
Done 673fabd